uncaught.js

```diff
@@ -248,13 +329,13 @@ function stringifyError(error, uncaughtType) {
     var d = new Date();
 
     return jsonStringify({
-        message: error.message,
-        type: error.type,
+        message: error && error.message,
```

Comment:
why were these safe guards necessary? it seems that you trust `error` is defined elsewhere in the code.